test_runner/regress/test_compute_ctl.py

```diff
@@ -240,7 +241,7 @@ def __exit__(self, _type, _value, _traceback):
         with self.pid_file:
             try:
                 os.kill(self.pid, signal.SIGTERM)
-            except os.OsError as e:
+            except OSError as e:
```

Comment:
I ended up adding that `os.` because my python lsp insisted wasn't finding OSError without it, even though I understood from the docs it should be "in the implicit prelude". Is this following some direct change of something else in this PR, like mypy checking more things?

Reply:
Yep `mypy` started to complain about it with `check_untyped_defs`:
```
test_runner/regress/test_compute_ctl.py:244: error: Module has no attribute "OsError"; maybe "OSError" or "_OnError"?
Found 1 error in 1 file (checked 103 source files)
```

Reply:
Also, I'm not sure what `os.OsError` is:
```
$ python3.9 -c "import os; print(os.OsError)"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: module 'os' has no attribute 'OsError'
```
```
$ python3.9 -c "import os; print(os.OSError)"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: module 'os' has no attribute 'OSError'
```
```
$ python3.9 -c "import os; print(OSError)"
<class 'OSError'>
```

Reply:
in last example you don't need to `import os`. `OSError` is a part of `builtins`